backends/v2/src/main.rs

```diff
@@ -36,6 +36,8 @@ struct Args {
     hostname: String,
     #[clap(default_value = "3000", long, short, env)]
     port: u16,
+    #[clap(default_value = "9000", long, short, env)]
+    prometheus_port: u16,
```

Comment:
I have created a separate PR for prometheus_port, will rebase once that is merged.